### ISSUE TITLE
build gradle failed cause duplicate value resource 'attr/font' with config ' '

### ISSUE THREAD

**Issue by yudikarma** [MuhammadArsalanChishti/Cuboid-Circle-Button#8]
Android resource compilation failed
Output:  /home/yudikarma/AndroidStudioProjects/Task107_imagesqilte/app/build/intermediates/incremental/mergeDebugResources/merged.dir/values/values.xml:1271: error: duplicate value for resource 'attr/font' with config ''.
/home/yudikarma/AndroidStudioProjects/Task107_imagesqilte/app/build/intermediates/incremental/mergeDebugResources/merged.dir/values/values.xml:1271: error: resource previously defined here.

Command: /home/yudikarma/.gradle/caches/transforms-1/files-1.1/aapt2-3.2.1-4818971-linux.jar/16f95533f32d0eb3ec6b15dd8253a625/aapt2-3.2.1-4818971-linux/aapt2 compile --legacy \
        -o \
        /home/yudikarma/AndroidStudioProjects/Task107_imagesqilte/app/build/intermediates/res/merged/debug \
        /home/yudikarma/AndroidStudioProjects/Task107_imagesqilte/app/build/intermediates/incremental/mergeDebugResources/merged.dir/values/values.xml
Daemon:  AAPT2 aapt2-3.2.1-4818971-linux Daemon #0

**Comment by jamshaid120** [comment]
i am also having the same issue. did you find the solution 
?

**Comment by NiDaAppDev** [comment]
Nope, unfortunately I couldn't find the solution for the specific
problem... But I did find an alternative option - creating the button
resource by yourself
Read more about it in here:
https://stackoverflow.com/questions/9884202/custom-circle-button

Then, in order to make the hight of it equal to the width (making it
perfect circle and not some oval), I found this:
https://stackoverflow.com/questions/9798392/imageview-have-height-match-width
I can't remember if it worked for me or not, so if it doesn't work for you,
go ahead and search for this problem in the internet, I remember it as a
not-to-hard to find the solution
Good luck!
